vs/windows.undocked.props

```diff
@@ -226,7 +226,7 @@
     </Link>
   </ItemDefinitionGroup>
   <!-- Enable ARM64X compilation -->
-  <PropertyGroup Condition="'$(Platform)'=='ARM64' AND '$(UndockedType)' != 'sys' AND '$(UndockedType)' != 'drvlib' AND '$(UndockedType)' != 'exe' AND '$(EnableArm64x)' == 'true'">
+  <PropertyGroup Condition="'$(Platform)'=='ARM64' AND '$(UndockedType)' != 'sys' AND '$(UndockedType)' != 'drvlib' AND '$(UndockedType)' != 'exe' AND '$(UndockdedBuildArm64x)' == 'true'">
```

Comment:
Should we get rid of the exclusion Conditions since this is now opt-in?

Reply:
I'd prefer to leave them in for now. We can simplify later if we find they are unnecessary.

Reply:
They're still needed to prevent targets that don't support ARM64X from being built that way.